R/functions__lazy.R

```diff
@@ -132,426 +142,283 @@ pl_implode = function(name) { # -> Expr
     unwrap("in pl$implode():")
 }
 
-#' pl$first
-#' @description  Depending on the input type this function does different things:
-#' @param column if dtype is:
-#' - Series: Take first value in `Series`
-#' - str: syntactic sugar for `pl.col(..).first()`
-#' - NULL: expression to take first column of a context.
-#'
-#'
-#' @keywords Expr_new
-#'
-#' @return Expr or first value of input Series
+#' Get the first value.
 #'
+#' This function has different behavior depending on arguments:
+#' - Missing -> Takes first column of a context.
+#' - Character vectors -> Syntactic sugar for `pl$col(...)$first()`.
+#' @param ... Characters indicating the column names.
+#' If missing (default), returns an expression to take the first column
+#' of the context instead.
```

Comment:
Can you add that regular expressions are supported too? Not just here but for the other functions too